lib/load.php

```diff
@@ -33,9 +33,18 @@
 
 
 // Register server-side code for individual blocks.
-foreach ( glob( dirname( __FILE__ ) . '/../block-library/*/index.php' ) as $block_logic ) {
-	require $block_logic;
-}
-foreach ( glob( dirname( __FILE__ ) . '/../packages/block-library/src/*/index.php' ) as $block_logic ) {
-	require $block_logic;
+$paths = array(
+	dirname( __FILE__ ) . '/../block-library/*/index.php',
+	dirname( __FILE__ ) . '/../packages/block-library/src/*/index.php',
+);
+
+foreach ( $paths as $path ) {
+	$block_logic_files = glob( $path );
+	if ( ! $block_logic_files ) {
```

Comment:
Might be worth an inline comment explaining the context here, since it won't otherwise be obvious to the future maintainer why this is necessary.